packages/api-key-service/src/functions/create-api-key.ts

```diff
@@ -21,7 +21,7 @@ export const createApiKey = httpHandler<ApiKeyDto, ApiKeyModel>({
       if (typeof customer === 'undefined' || !customer.developerId) {
         throw new Error('Unauthorised')
       }
-    } catch (e) {
+    } catch (e: any) {
```

Comment:
Rather than `any` you could do 

`const error = e as Error` and then pass `error.message` to the exception handler? Maybe a bit nicer?